odoo/fields.py

```diff
@@ -319,13 +326,15 @@ def _get_attrs(self, model, name):
             if not (attrs['store'] and not attrs.get('readonly', True)):
                 attrs['copy'] = attrs.get('copy', False)
             attrs['readonly'] = attrs.get('readonly', not attrs.get('inverse'))
+            attrs['pre_compute'] = attrs.get('pre_compute', model._pre_compute)
```

Comment:
This makes unit testing more difficult.  Instead, add an explicit test on `model._pre_compute` in the auxiliary methods of `create`.  This flag must eventually disappear (to be discussed).